index.html

```diff
@@ -0,0 +1,110 @@
+  </head>
+  <body>
+    <section class="header">
+      <a class="name" href="assets/images/Vahan.jpg"
+        ><p>Vahan Khachvankian</p></a
+      >
+      <nav class="burger">
+        <a href="#"><img src="assets/images/lines-56763.png" alt="logo" /></a>
+      </nav>
+    </section>
+
+    <section class="main">
+      <div class="title">
+        <h1>
+          Hey There. <br />
+          I'm Vahan
+        </h1>
+      </div>
+      <div class="head">
+        <p class="pretext">I'am a Software Developer</p>
+      </div>
+      <div class="content">
+        <p class="text">
+          I can help you build a product, feature or website. Look through of
+          some my work and experience! If you like what you see and have a
+          project you need coded, don't hesitate to contact me.
+        </p>
+      </div>
+
+      <nav class="navigator">
+        <ul id="menu">
+          <li>
+            <a
+              href="https://github.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="github height"
+                src="assets/images/github.svg"
+                alt="github logo"
+            /></a>
+          </li>
+          <li class="bg height">
+            <a
+              href="https://www.vectorstock.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="vector-one height"
+                src="assets/images/Vector.png"
+                alt="vector-one" />
+              <img
+                class="vector-two height"
+                src="assets/images/VectorN.png"
+                alt="vector-two"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://angel.co/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="angel height"
+                src="assets/images/angellist.svg"
+                alt="angel logo"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://twitter.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="twitter height"
+                src="assets/images/twitter.svg"
+                alt="twitter logo"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://medium.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                class="medium height"
+                src="assets/images/medium.svg"
+                alt="medium logo"
+            /></a>
+          </li>
+        </ul>
+        <div class="background">
+          <img class="bottom" src="assets/images/VectorZet.png" alt="bottom" />
+        </div>
+      </nav>
+    </section>
```

Comment:
- [ ] Kindly fix the overflow on the left side of your page🙏 for a better user interface😉
- [ ] Also, please make sure your design matches your preferred template as much as possible😎
- Your design
![screenshot (38)](https://user-images.githubusercontent.com/44624138/143940850-ddd1d24b-880d-4b12-ad83-a4a241324d5e.png)
- Figma design
![screenshot (39)](https://user-images.githubusercontent.com/44624138/143940908-2fd20a63-f7dd-48c7-8c29-745b25ffabb6.png)

Reply:
Hi, Aziz,
Could you kindly clarify, what do you mean by saying "the overflow on the left side" of my page? 
I can't pinpoint it for sure. 
Sorry to bother you, but I still have one try not to fly out.

Reply:
Done. Add body properties.